cli/crates/server/src/bridge/server.rs

```diff
@@ -116,6 +117,7 @@ pub async fn start(port: u16, worker_port: u16, event_bus: Sender<Event>) -> Res
     let pool = SqlitePoolOptions::new().connect(&db_url).await?;
 
     query(PREPARE).execute(&pool).await?;
+    let _ = query(MIGRATION_001).execute(&pool).await; // Ignore error when `owned_by` column already exists.
```

Comment:
Since this may error in other ways, we may want to check for the column beforehand / check for the specific error.
We could also consider a breaking change and instruct people to reset beforehand since we're not in public beta yet

Reply:
> We could track migrations in a separate table
That should work and allow us to handle errors directly